extensions/whox.md

```diff
@@ -0,0 +1,96 @@
+---
+title: WHOX
+layout: spec
+copyrights:
+  -
+    name: "Simon Ser"
+    period: "2021"
+    email: "contact@emersion.fr"
+---
+
+## Introduction
+
+The `WHO` command allows clients to obtain information about other clients.
+However, `WHO` doesn't allow clients to request extra data (e.g. account names)
```

Comment:
Bikeshedding the language a little here, I think this sentence might read better as:

>The `WHOX` extension allows clients to request additional fields in the `WHO` response (e.g. account name) or omit default fields (e.g. username and hostname).

Reply:
That's better! Applied, thanks!